microsoft-edge/webdriver-chromium/ie-mode.md

```diff
@@ -0,0 +1,222 @@
+---
+# Use Internet Explorer Driver to automate IE mode in Microsoft Edge
+
+Internet Explorer (IE) mode in Microsoft Edge is a feature for organizations that still need Internet Explorer 11 for backward compatibility for legacy websites or apps.  To learn more about IE mode, read [What is Internet Explorer (IE) mode?](/deployedge/edge-ie-mode)
+<!-- = https://docs.microsoft.com/deployedge/edge-ie-mode -->
+
+Internet Explorer 11 will no longer be supported on certain versions of Windows 10 starting **June 15, 2022**. For more information about ending support for Internet Explorer 11, read [Internet Explorer 11 desktop application ending support for certain operating systems](/lifecycle/announcements/internet-explorer-11-end-of-support).
+<!-- = https://docs.microsoft.com/lifecycle/announcements/internet-explorer-11-end-of-support -->
+
+Organizations with business-critical legacy websites or apps may need to test their content in IE mode in Microsoft Edge.  The following sections describe how to get started with Internet Explorer Driver (IEDriver) to automate IE mode in Microsoft Edge.
+
+
+<!-- ====================================================================== -->
+## Download Internet Explorer Driver (IEDriver)
+
+To begin automating tests in IE mode in Microsoft Edge, [download IEDriver](https://www.selenium.dev/downloads/).  Make sure that the version of IEDriver that you download is `4.0.0.0` or greater.
+
+:::image type="content" source="./media/iedriver-install.msft.png" alt-text="The IEDriver section of Downloads page for Selenium." lightbox="./media/iedriver-install.msft.png":::
+
+
+<!-- ====================================================================== -->
+## Automate IE mode in Microsoft Edge
+
+The following sections walk you through using Selenium to automate IE mode in Microsoft Edge.
+
+> [!NOTE]
+> This article provides instructions for using the Selenium framework, but you can use any library, framework, and programming language that supports WebDriver.  To accomplish the same tasks using another framework, consult the documentation for your framework of choice.
+
+To launch Microsoft Edge in IE mode with IEDriver:
+
+1.  Create an `InternetExplorerDriverService`.
+
+1.  Define `InternetExplorerOptions` with additional capabilities that point to the Microsoft Edge browser.
+
+1.  Start an instance of `InternetExplorerDriver` and pass it `InternetExplorerDriverService` and `InternetExplorerOptions`.  IEDriver launches Microsoft Edge and then loads your web content in IE mode.
+
+The next section shows the complete sample, and then the subsequent sections focus on each of the main steps that are listed above.
+
+
+<!-- ====================================================================== -->
+## The complete sample
+
+The following sample launches Microsoft Edge in IE mode, navigates to [bing.com](https://www.bing.com/) and searches for "WebDriver".
+
+### [C#](#tab/c-sharp/)
+
+<a id="drive-ie-mode-entire-sample"></a>
```

Comment:
I doubt these <a> tags are used at all.  Maybe they should be removed from the repo (37 hits in 4 files).
They aren't mentioned in Docs Contributor Guide's article:
https://review.docs.microsoft.com/en-us/help/contribute/tabbed-conceptual-overview?branch=master&tabs=windows%2Cazure-cli